docs/modules/user_guide/tree.rst

```diff
@@ -0,0 +1,116 @@
+the most common apporaches used for unsupervised classification are 
+K-Means clustering and Mixture Modeling, which use the Expectation 
+Maximization Algorithm.
+
+In clustering settings, there may be no natural notion of “true”
+class-labels; the efficacy of the clustering scheme is often measured 
+in such cases by the economy in description length attained
+by a two-step description of the objects by first describing the 
+attributes common to the clusters and then describing the differential
+attributes of each object within the cluster. k-Means Clustering
+and Mixture Modeling using the Expectation Maximization Algorithm 
+are examples of techniques used for unsupervised classification.
+
+Another way to use unsupervised learning with decision trees is to use 
+clustering to pre-process the data before training a supervised 
+decision tree. This can help identify patterns in the data and reduce 
+noise, which can improve the performance of the decision tree.
+
+In summary, "unsupervised decision trees" could refer to either 
+clustering-based decision trees or decision trees trained on 
+pre-processed data using unsupervised learning techniques.
+
+
+.. _tree_cluster:
+
+Cluster criteria
+----------------
+
+Clustering criterion for :class:`sktree.tree.UnsupervisedDecisionTree` is a technique that combines clustering 
+and decision tree algorithms to produce a set of decision rules. The clustering 
+criterion can be used to identify clusters of data that have similar 
+characteristics, and then construct decision rules based on the clustering 
+results.
+
+One common clustering criterion for decision trees is the :class:`sklearn.cluster.KMeans` algorithm. 
+The :class:`sklearn.cluster.KMeans` algorithm partitions the data into K clusters based on the 
+distance between the data points. The algorithm then constructs a decision 
+tree on each cluster, using the same process as a standard decision tree 
+algorithm.
+
+Another clustering criterion for decision trees is the :class:`sklearn.cluster.AgglomerativeClustering`
+algorithm. Agglomerative Clustering is a bottom-up clustering method that starts 
+with each data point as a cluster and then iteratively merges the closest 
+pairs of clusters until all the data points are in a single cluster. 
+The resulting tree structure can be used to construct a decision tree by 
+selecting a threshold to cut the tree at a particular level.
+
+The advantage of using clustering criteria in decision trees is that 
+it can help identify subgroups of data with different characteristics, 
+which can lead to more accurate and interpretable models. However, the 
+choice of clustering criterion and the number of clusters can significantly 
+affect the performance of the decision tree. Therefore, it is important to 
+carefully evaluate different clustering criteria and parameter settings to 
+find the best approach for a particular dataset.
```

Comment:
This entire section reads as if tho the TwoMeans and FastBIC objective is used to perform clustering with Means and Agglomerative Clustering.